lib/apiservers/engine/backends/image.go

```diff
@@ -149,14 +149,14 @@ func (i *Image) PullImage(ctx context.Context, ref reference.Named, metaHeaders
 	err := cmd.Start()
 
 	if err != nil {
-		log.Printf("Error starting %s - %s\n", Imagec, err)
+		log.Debugf("Error starting %s - %s\n", Imagec, err)
```

Comment:
This and the one below should be errors - this is in the context of the appliance logs, so logging recoverable errors as errors is okay. We do need a much better contextual logging story.